index.html

```diff
@@ -0,0 +1,243 @@
+    <link rel="stylesheet" href="style.css">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap');
+      </style>
+      <script
+      src="https://kit.fontawesome.com/675259b560.js" crossorigin="anonymous">
+    </script>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+            <section id="home">
+                <nav id="navbar" class="open">
+                    <div class="container">
+                        <a href='#' class="logo">My Logo</a>
+                        <div class="boton"><i class="fa-solid fa-bars fa-2xs"></i></div>
+                    </div>
+                </nav>
+            </section>
+    </header>
+    <main>
+        <section class="portfolio">
+            <div>
+                <h1>Hey There. I'm Jonathan.</h1>
+            </div>
+            <div>  
+                <h2>I'm a Software Developer</h2>
+            </div>
+            <div>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        dont hesitate to contact me.</p>
+            </div>
+            
+            <div class="icons">
+                <a href="">
+                    <img src="Assets/Github.svg" alt="Github">
+                </a>
+                <a href="">
+                    <img src="Assets/Linkedin.svg" alt="Linkedin">
+                </a>
+                <a href="">
+                    <img src="Assets/Icon - Angelist.svg" alt="Angelist">
+                </a>
+                <a href="">
+                    <img src="Assets/Twitter.svg" alt="Twitter">
+                </a>
+                <a href="">
+                    <img src="Assets/Medium.svg" alt="Medium">
+                </a>
+            </div>
+        </section>
+        <section id="experience">
+            
+            <h2 class="title">My Recent Works</h2>        
+            <img class="vector" src="Assets/Vector.svg" alt="">        
+            <div>
+                <img  class="placeholder" src="Assets/Img Placeholder.svg" alt="">
+                
+                <div>
+                    <h2 class="work">Multi-Post Stories</h2>
+                </div>
+                <div>
+                    <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+                <div class="teclast">
+                    <div class="css-button">CSS</div>
+                    <div class="html-button">HTML</div>
+                    <div class="boot-button">Bootstrap</div>
+                    <div class="ruby-button">Ruby</div>
+                </div>
```

Comment:
- [ ] Please 🙏🏻  use the `<ul>` tag over `div` to enhance the professionalism and semantic correctness of your code, use the `<ul>` tag as a container and place each social media icon link within an individual **li** tag. This approach not only improves the structure of your *HTML* but also benefits **visually-impaired** individuals, ensuring a better user experience. Avoid using the `<div>` tag solely for decorative purposes, as it lacks semantic meaning.

   - Once you make the necessary changes, remember to add appropriate styles to your CSS file. Implementing 
     **Flexbox** can help you properly align the items within the **ul** container. Utilizing `Flexbox` provides a flexible 
     and responsive layout, ensuring consistent presentation across different screen sizes and devices.

   - It is crucial to always close your tags with the correct tag name. This practice becomes even more significant as your code scales or when you add more features in the future. By diligently closing your tags, you avoid 
     potential errors that may arise due to small oversights. Maintaining proper tag closure ensures code readability, makes debugging easier, and helps maintain the integrity of your markup.

- [ ] Kindly stick with the *Figma* design as closely as possible, it seems like your font family is not matching in your work using the `Inter` font family. Please have a look at the below comparison table to understand the problem in a better way through visualization.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-06 13-14-41](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/aa9935b2-4d54-4722-bbce-0ce9913e828b)


</td>

<td>

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/5b62ae2f-fba1-426f-b16c-31830c112d6a)


</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)